CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -91,5 +90,38 @@ public void CanRemindAUser()
 
             ThenMessageHasBeenPostedToSlack("U172L982");
         }
+
+        [Test]
+        public void CanGetUsersFromHarvest()
```

Comment:
At some point we should refactor this acceptance test so that it's in the Given, When, Then format just like our previous acceptance test.

Reply:
at some point, we will! ;)